completions/bash/multipass

```diff
@@ -193,6 +193,9 @@ _multipass_complete()
     fi
 
     case "${cmd}" in
+        "exec")
+            opts="${opts} --working-dir"
```

Comment:
You have the full word there, and also the short version:

```suggestion
            opts="${opts} --working-directory -w"
```

Reply:
Hey @Saviq, thanks for the comment! I'll correct the typo but, as for the short version, I just followed what we're doing for completions (i.e., we just complete with long options).